content/tracing/setup/javascript.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Tracing JavaScript Applications
+kind: Documentation
+aliases:
+- /tracing/javascript/
+- /tracing/languages/javascript/
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-js"
+  tag: "Github"
+  text: Source code
+- link: "https://datadog.github.io/dd-trace-js"
+  tag: "Documentation"
+  text: API documentation
+- link: "tracing/visualization/"
+  tag: "Use the APM UI"
+  text: "Explore your services, resources and traces"
+---
+
+<div class="alert alert-warning">
+JavaScript APM is currently in <strong>beta</strong>.
+</div>
+
+## Getting started
```

Comment:
The order of this page is a little funny to me (for example having config options after manual instrumentation seems strange).  Could we mirror how we order this info (and language where applicable) the same as on https://docs.datadoghq.com/tracing/setup/java/.

Could we also add what versions of Node are supported at the top?  I don't see that elsewhere in the docs.